applications/zklogin-Reclaim-protocol.md

```diff
@@ -0,0 +1,299 @@
+
+Reclaim protocol will allow Dapps within the Polkadot ecosystem to provide zk login for its users. It will enable Dapp users to log into any website and generate a zkproof of their login that can be verified onchain. Reclaim Protocol makes https traffic verifiable using Zero-Knowledge Proofs, enabling users to generate verifiable credentials from any of their online user profiles. This unlocks unlimited possibilities as no APIs are required or no changes to be made to the websites to extract private user data, while guaranteeing data integrity. Web2 user data which was elusive to Web3 till now will be available to builders across the Polkadot ecosystem. Reclaim empowers developers to make Web2 composable. This opens up opportunities for a new wave of applications that can support zklogin, proof of personhood, and bot protection. 
+
+Additionally, after speaking with the members of the Polkadot Foundation, we have been able to identify zklogin and [epassport](https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/epassport-zk-validation.md) to be important use cases for Dapp builders and users that can be implemented uniquely through Reclaim Protocol. Through Reclaim, users can not only validate their login credentials but also provide verifiable proofs of various personal attributes and activities. **For instance, they can prove the number of rides taken on Uber, expenditures on Amazon, or their scores on Chess.com, thereby establishing a robust proof of personhood which is instrumental in combating fraudulent actors and bots. Such proof of personhood can now be made available on-chain via Reclaim protocol within the Polkadot ecosystem.**
+
+We are a small team (20+ members) with finite resources and want to collaborate with ecosystems that demonstrate a genuine commitment to enable use cases that are likely to scale to a billion users. Our team comprises primarily engineers who have previously worked at Meta, Microsoft, Google and have contributed to various web3 open source projects. Given Polkadot's strong emphasis on streamlining user onboarding,  user experience, and implementing bot protection, coupled with our team's expertise and background, we believe that our team is one of the most suited to realise Polkadot's vision of building zk login and implementing Proof of Personhood for Dapps on top of Polkadot.
+
+
+
+### Project Details
+
+[zk Login Overview](https://questbook.notion.site/Implementing-zklogin-and-Proof-of-Personhood-using-Reclaim-Protocol-6a1d31a7d70c46c482682742abbbe0cb#:~:text=using%20Reclaim%20Protocol-,zk%20Login%20Overview,-Reclaim%20Protocol%20Overview)
```

Comment:
Absolutely no reason for the request to hit polkadot

What is the "social login"?  It's never defined elsewhere.

Reply:
In order to verify zk proofs generated by users using Reclaim protocol, Dapp smart contracts need to interact with a verification smart contract, which is an integral part of Reclaim’s SDK. This interaction is crucial for verifying proofs generated through Reclaim, enabling Dapp developers to make checks of users’ identity and reputation in their smart contracts. The request made to Polkadot is specifically for this verification process as the contracts will live on chain.

The social login here refers to the websites such as Google, Facebook, Instagram that the user will log into to generate zk proofs of their data ownership related to the considered website. We currently support over 250 providers, including Steam, Github, [Aadhar](https://uidai.gov.in/en/), Spotify, etc., chosen based on existing Dapp developers' needs. We're also open to adding more providers as per the requirements of Dapp developers within the Polkadot ecosystem.